x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.tsx

```diff
@@ -516,6 +520,21 @@ export function DimensionEditor(props: DimensionEditorProps) {
       };
     }
   );
+  const softRestrictedSideNavItems = sideNavItems.filter(
+    (navItem) =>
+      referencedField &&
+      referencedField.softRestrictions &&
+      referencedField.softRestrictions[navItem.id!]
+  );
+  if (softRestrictedSideNavItems.length > 0) {
+    sideNavItems = sideNavItems.filter(
+      (navItem) =>
+        !referencedField ||
+        !referencedField.softRestrictions ||
+        !referencedField.softRestrictions[navItem.id!]
```

Comment:
>Why do we allow these to be selected at all?

Because the time range can change and they can suddenly work. The time range dictates if the indexes that are returned are downsampled or not